docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,3685 @@
+        </p>
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+    
+
+
+
+
+We can see what made the network decide that is is a negative example.
+
+Another argument ``counterfactual`` (default ``False``) highlights the
+counter-evidence for a class, or what makes the score "go down" (set to
+``True`` to enable). This is mentioned in the Grad-CAM paper
+(https://arxiv.org/abs/1610.02391).
```

Comment:
Maybe would be good to add that for binary classification, this is the same as inverting the classes.